web/backend/src/controllers/dela.ts

```diff
@@ -263,6 +238,30 @@ delaRouter.use('/*', (req, res) => {
   }
 
   const bodyData = req.body;
+
+  // special case for voting
+  const match = req.baseUrl.match('/api/evoting/forms/(.*)/vote');
+  if (match) {
+    if (!req.session.userId) {
+      res.status(401).send('Authentication required!');
+      return;
+    }
+    if (!isAuthorized(req.session.userId, match[1], PERMISSIONS.ACTIONS.VOTE)) {
+      res.status(400).send('Unauthorized');
+      return;
+    }
+
+    // We must set the UserID to know who this ballot is associated to. This is
+    // only needed to allow users to cast multiple ballots, where only the last
+    // ballot is taken into account. To preserve anonymity, the web-backend could
+    // translate UserIDs to another random ID.
+    // bodyData.UserID = req.session.userId.toString();
+
+    // DEBUG: this is only for debugging and needs to be replaced before production
+    console.warn('DEV CODE - randomizing the SCIPER ID to allow for unlimited votes');
+    bodyData.UserID = makeid(10);
```

Comment:
can we use a global env. variable to define if we're in debug or not, instead of commenting out ? I'm afraid something like this will make its way back into production someday when someone's looking in the other direction :-)

Reply:
Definitely better to use a global variable. One of the problems is that we cannot use `NODE_ENV === 'development'`, as we currently need to run the frontend in development mode due to the proxying.